source/tutorials/packaging-existing-software.md

```diff
@@ -0,0 +1,567 @@
+---
+myst:
+  html_meta:
+    "description lang=en": "Packaging Existing Software With Nix"
+    "keywords": "Nix, packaging"
+---
+
+
+(packaging-existing-software)=
+
+
+# Packaging Existing Software With Nix
+
+One of Nix's primary use-cases is in addressing common difficulties encountered while packaging software, like *managing dependencies*.
+
+In the long term, Nix helps tremendously in alleviating that stress, but when *first* (re)packaging existing software with Nix, it's common to encounter missing dependencies preventing builds from succeeding.
```

Comment:
Good expectation management. Some thoughts
- Flaws also include downloading from the internet
- Discovering dependencies by letting the build fail may be a good practice. We don't want to add unused dependencies.